### PR TITLE
L2-740 and L2-736: Fix e2e tests

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Fix e2e tests to be able to enable them again in CI pipeline.

# Changes

* Wait for an element to be clickable before changing the value.
* Update readme.
* Edit setup script.
* Remove unused "test.ts".
* Add screenshots to .gitignore

# Tests

Fix and enable some e2e tests.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@bitdivine fixed and uncommented e2e tests

I also did some cleaning in the repo and fixed some lines in the setup script.

File: e2e-tests/components/icp.ts [hunk 1]
```diff
@@ -27,6 +27,8 @@ export class Icp extends MyNavigator {
       "Get the field to enter how much ICP we want",
       { timeout: 30_000 }
     );
+    // Not intially enabled to be changed.
```
Comment (peterpeterparker) [inline e2e-tests/components/icp.ts:30]: typo "intially"

File: e2e-tests/specs/user-V01-open-proposals-are-shown.e2e.ts [hunk 2]
```diff
@@ -50,10 +50,9 @@ describe("Makes a proposal and verifies that it is shown", () => {
       proposalMetadataSelector,
       "Verifying that the proposal detail is for the correct proposal"
     );
-    // TODO(L2-736): This test is failing for unknown reason with the new design
-    // await navigator.click(
-    //   ProposalsTab.HEADER_BACK_SELECTOR,
-    //   "Go back to proposal list"
-    // );
+    await navigator.click(
```
Comment (peterpeterparker) [inline e2e-tests/specs/user-V01-open-proposals-are-shown.e2e.ts:53]: did you find the root cause of the issue in the ci?

Comment (lmuntaner) [reply e2e-tests/specs/user-V01-open-proposals-are-shown.e2e.ts:53]: Not for this one.

I enabled it and it worked...

File: e2e-tests/components/icp.ts (hunk 1 above)
Comment (lmuntaner) [reply e2e-tests/components/icp.ts:30]: thanks!

File: e2e-tests/specs/user-V01-open-proposals-are-shown.e2e.ts (hunk 2 above)
Comment (lmuntaner) [reply e2e-tests/specs/user-V01-open-proposals-are-shown.e2e.ts:53]: I remember. Very weird indeed.

Even the failing test I fixed was also weird. It took me ages to get it right.

File: scripts/setup [hunk 3]
```diff
@@ -206,8 +206,8 @@ command -v xz || "install_xz_$OS"
 have_correct_dfx || install_dfx
 command -v gtar || tar --help | grep GNU >/dev/null || "install_tar_$OS"
 command -v cmake || "install_cmake_$OS"
-command -v ic-cdk-optimizer || install_ic_cdk_optimizer
-command -v ic-admin || "install_ic_admin_$OS"
+command -V ic-cdk-optimizer || install_ic_cdk_optimizer
```
Comment (peterpeterparker) [inline scripts/setup:209]: not a command line expert, is that an on purpose change?

above `cmake` still uses `-v` and these two are using `-V`

File: e2e-tests/specs/user-V01-open-proposals-are-shown.e2e.ts (hunk 2 above)
Comment (peterpeterparker) [reply e2e-tests/specs/user-V01-open-proposals-are-shown.e2e.ts:53]: Let's go on then and we shall keep an eye on it

File: scripts/setup (hunk 3 above)
Comment (lmuntaner) [reply scripts/setup:209]: I am not a cli expert, but those two "ic-cdk-optimizer" and "ic-admin" require a `-V` with uppercase "V", while the others a lowercase "v". I don't know why.

```
command -v cmake
```

This is the docs when you print the help:

```
   -V, --version
            Print version informatio
```

Comment (peterpeterparker) [reply scripts/setup:209]: oki doki thanks for the feedback

**Review by peterpeterparker — APPROVED** [review]
LGTM

As discussed, we keep an eye on the "weird test" 😉

File: scripts/setup (hunk 3 above)
Comment (bitdivine) [reply scripts/setup:209]: Technically either should work; if `-V` works and `-v` doesn't something weird has happened and I'd love to know the details.  But given that all we care about is "is it installed?" either is valid.

Comment (bitdivine) [reply scripts/setup:209]: Example.  Both succeeding:
```
max@sinkpad:/ (25)$ command -v sns
/home/max/bin/sns
max@sinkpad:/ (26)$ command -V sns
sns is /home/max/bin/sns
max@sinkpad:/ (29)$ 
```
or:
```
max@sinkpad:/ (1:34)$ command -v ic-cdk-optimizer ; echo $?
/home/max/.cargo-home/bin/ic-cdk-optimizer
0
max@sinkpad:/ (2:43)$ command -V ic-cdk-optimizer ; echo $?
ic-cdk-optimizer is /home/max/.cargo-home/bin/ic-cdk-optimizer
0
```
Both failing, due to using an incorrect name:
```
max@sinkpad:/ (1:16)$ command -V install_ic_cdk_optimizer ; echo $?
bash: command: install_ic_cdk_optimizer: not found
1
max@sinkpad:/ (1:28)$ command -v install_ic_cdk_optimizer ; echo $?
1
```
Note: The 0 or 1 is the return code, just to prove that it's succeeding or failing.

Comment (lmuntaner) [reply scripts/setup:209]: Weird, it wasn't doing the same for me when I tried it. It wasn't installing `ic-admin`, that's when I checked.

But yes, now it's working, I get the same.

**Pull request merged**